controllers/configurationpolicy_controller.go

```diff
@@ -68,6 +70,19 @@ var config *rest.Config
 // NamespaceWatched defines which namespace we can watch for the GRC policies and ignore others
 var NamespaceWatched string
 
+var evalLoopHistogram = prometheus.NewHistogram(
+	prometheus.HistogramOpts{
+		Name:    "config_policies_evaluation_duration_seconds",
+		Help:    "The seconds that it takes to evaluate all configuration policies on the cluster",
+		Buckets: []float64{1, 3, 9, 10.5, 15, 30, 60, 90, 120, 180, 300, 450, 600},
```

Comment:
I think these buckets are a good starting point. I think we need to use the metric in some tests and see which ones are useful, or where we might want more data. And I think I'd want us to re-visit this before a release, since changing it will cause incompatible data between versions.

Eg, we might find that the buckets past 60 aren't worth the additional time-series, and we might want more below 10 in order to more accurately estimate how many more policies we could add before hitting saturation (at 10s).